framework/src/org/checkerframework/framework/qual/AnnotatedFor.java

```diff
@@ -32,7 +35,8 @@
  *
  * @checker_framework.manual #compiling-libraries Compiling partially-annotated libraries
  */
-@Target({TYPE, METHOD, CONSTRUCTOR}) // permitting on PACKAGE would be too error-prone
+@Target({TYPE, METHOD, CONSTRUCTOR, PACKAGE})
```

Comment:
I am curious - is this change being made in order to support a particular scenario?